lib/rails_param/validator.rb

```diff
@@ -0,0 +1,40 @@
+module RailsParam
+  module Param
+    class Validator
+      attr_reader :parameter, :name, :options, :value
+
+      def initialize(parameter)
+        @parameter = parameter
+        @name = parameter.name
+        @options = parameter.options
+        @value = parameter.value
+      end
+
+      def validate!
+        options.each_key do |key|
```

Comment:
Some of the keys don't really require validation. Do you think it would makes to have a way of skipping certain keys? Like `default`, `transform`, etc?